### PR TITLE
Add outline param

### PR DESCRIPTION
**Pull request by talyguryn**

### CODE REVIEW

**Comment by iwsfg** [comment]
Sorry, I don't think it's a good idea to add this to the API. If we allow controlling `stroke` we may as well add options for `stroke-width`, `stroke-opacity`, `stroke-dasharray`, `stroke-dashoffset` and others.

If desired same result could be achieved in post processing step with replace by a simple regular expression or using HTML/SVG parser. More importantly, it doesn't contribute to the main purpose of this package: tracing. I guess same could be said about fill color though.

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ A NodeJS-compatible fork of [Potrace in JavaScript][potrace-by-kilobtye] with so
 Install
 
 ```sh
-npm install potrace
+npm install https://github.com/talyguryn/node-potrace
```
Comment (iwsfg) [inline README.md]: :unamused:

Comment (talyguryn) [reply README.md]: oh, sorry 😬

**Comment by talyguryn** [comment]
I need to create contour as a cutline for image to create stickers. So I need to use only stroke because fill gives a colored area between two lines. It gives problems when I process images before printing and cutting. I think it could be useful for anyone who wants to get only contour of image.

**Pull request closed without merging**